assets/src/edit-story/app/uploader/utils/test/error.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import createError from '../error';
+
+describe('createError', () => {
+  it('should return an error', () => {
+    expect(
+      createError('testing', 'test.jpg', 'hello there').toString()
+    ).toStrictEqual('testing: hello there');
```

Comment:
This test really just tests [`Error.prototype.toString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString).

Checking the type and the error's properties might be more useful.

Reply:
Error objects are a special kind of object are hard to test for as the test normal error out when one is generated.

This test proves that a message is set, which is the most important part of the helper.